### ISSUE TITLE
nixpkgs haskell dependency brittleness checker

### ISSUE THREAD

**Issue by silky** [silky/ideas#704]
for a given haskell lib, check all the dependencies on nixpkgs and check which ones are broken

do it as a visualisation per package

could be a nice thing to add to, say, flora.pm ( cc @tchoutri )

**Comment by tchoutri** [comment]
yeah that could be nice to have. A monster of work but nice to have.
